contracts/trading/TradingModule.sol

```diff
@@ -246,9 +246,22 @@ contract TradingModule is Initializable, UUPSUpgradeable, ITradingModule {
         decimals = RATE_DECIMALS;
     }
 
+    function _getFlag(uint32 flags, uint16 flagID) private pure returns (bool) {
```

Comment:
Why is flagID in uint16 here? Why not uint32?

Reply:
fixed